CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.21)
+
+project(c-octo VERSION 1.2 LANGUAGES C)
+
+if(NOT WIN32)
+  set(INSTALLDIR bin)
```

Comment:
CMake defaults to /usr/local for UNIX platforms, so only the suffix is needed to match the behaviour of the Makefile. See https://cmake.org/cmake/help/v3.21/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX